Altis_Life.Altis/stringtable.xml

```diff
@@ -9265,7 +9265,7 @@
             <Polish>Pojazd jest zbyt daleko lub jesteś w!</Polish>
             <Russian></Russian>
             <Portuguese>O veículo está muito longe ou você está nele!</Portuguese>
-            <French>Le vehicule est trop loin, ou tu es dedans !</French>
+            <French>Le vehicule est peut etre trop loin, ou tu es dedans !</French>
```

Comment:
Without dash (-) : "peut être" = "can be"
Exact should be "peut-être" = "perhaps/maybe"